### DISCUSSION TITLE
Support for Microsoft Office and PDF files for windows

### DISCUSSION THREAD

**Discussion by randubin** [qilingframework/qiling#1082] [Ideas]
Hi All
Is it possible to extend and emulate Office & PDF files?
What are the steps needed to support it? I will be happy to contribute.

**Comment by wtdcode** [comment]
AFAIK, they are not executable files. What would you like to do exactly?

**Reply by randubin** [reply]
Emulating office and PDF files is not new but has high benefits. Winword.exe is running the ".doc" file, for example. Monitoring the behavior is interesting for malware detection.

**Reply by wtdcode** [reply]
So you are looking for emulating Office software? If so, you may start on contributing Windows emulation.
